### PR TITLE
Fix alumni page

### LINKED ISSUE #10: Update the alum page.

### PR DESCRIPTION
**Pull request by icco**
Lets fix this thing and make it pretty. I also updated this repo to be more in line with modern jekyll standards.

Who has cplug.org permissions still?

Fixes #10.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 
-Development
------------
+## Development
 
- 1. `gem install jekyll jekyll-redirect-from jekyll-paginate redcarpet`
- 2. `jekyll serve --watch`
- 3. start editing files
+ 1. `gem install bundler`
+ 2. `bundle install`
```
Comment (coyotebush) [inline README.md:8]: :heart: confirmed working (once I switched to using ruby2.3)

File: about/alums.md [hunk 2]
```diff
@@ -1,214 +1,228 @@
 ---
+
```
Comment (coyotebush) [inline about/alums.md]: Why blank lines in the [front matter](https://jekyllrb.com/docs/frontmatter/)?

File: about/alums.md [hunk 3]
```diff
@@ -1,214 +1,228 @@
...
 
-**Kevin Worth** (aka ceeKay) &#8211; 2006
+These are our distinguished alumni. Most hang out in the IRC channel, so if you want to know what it is like working at their companies, or get some real advice from the real world, feel free to ask.
+
+Note that most of them idle in the channel while at work, so your response (if you get one) could take hours or days.
+
+If you're an alumni and would like to be included here, please file a pull request on [github.com/CPLUG/cplug.org](https://github.com/CPLUG/cplug.org).
+
+* toc
+{:toc}
```
Comment (coyotebush) [inline about/alums.md:15]: awesome and :tophat: fancy!

File: about/alums.md [hunk 4]
```diff
@@ -1,214 +1,228 @@
...
- [3]: http://ruby-lang.org/
\ No newline at end of file
+ [3]: http://ruby-lang.org/
+
+
+## Nat Welch (icco) -- 2011
+
+### Current Job Situation
+
+I am currently working in Brooklyn on the Hillary Clinton campaign as a Senior Site Reliability Engineer. My resume is at [natwelch.com/resume](http://natwelch.com/resume), but since graduating I've worked at a few startups, Google and attended the [Recurse Center](http://recurse.com). You can stalk me at http://natwelch.com.
```
Comment (coyotebush) [inline about/alums.md]: link the last URL too?

File: about/alums.md [hunk 5]
```diff
@@ -1,214 +1,228 @@
...
-**Current Distribution: OSX, Ubuntu**
+### Current Distribution: OSX, Ubuntu
 
 i develop on OSX 'cause i need Adobe and Microsoft apps. running mostly Ubuntu on production servers, though i've used a lot of RHEL/CentOS in the past.
 
-**Other Tidbits:**
+### Other Tidbits
 
-i walked into a CPLUG event some time in 2006 or 2007 and had someone help me install Linux on a half-broken Dell laptop that my then-girlfriend had handed me down. i spent the next several months stumbling through admin'ing the machine, eventually even getting the NIC card working. this turned out to be the single most valuable learning experience i had in my academic years, and provided an invaluable foothold when starting my career. thanks.
+I walked into a CPLUG event some time in 2006 or 2007 and had someone help me install Linux on a half-broken Dell laptop that my then-girlfriend had handed me down. i spent the next several months stumbling through admin'ing the machine, eventually even getting the NIC card working. this turned out to be the single most valuable learning experience i had in my academic years, and provided an invaluable foothold when starting my career. thanks.
```
Comment (coyotebush) [inline about/alums.md]: Let's leave this consistent with the original and the other sentences.

File: about/alums.md [hunk 6]
```diff
@@ -1,214 +1,228 @@
...
-**Other Tidbits  
-**  
 Window Manager: wmii ([http://suckless.org][1]) &#8211; A tiling window manager&#8230; a little different from the norm, but it is so fast and works so well once you get used to the concept.
 
-Web comic: ([http://xkcd.com][2]) &#8211; If you don&#8217;t read this comic, you&#8230; well, you should read this comic.
+Web comic: ([http://xkcd.com][2]) &#8211; If you don't read this comic, you&#8230; well, you should read this comic.
 
 Language: Ruby ([http://ruby-lang.org][3])- This language is actually FUN to program in. I kid you not.
 
-\___\___\___\___\___\___\___\___\____
```
Comment (coyotebush) [inline about/alums.md:33]: Yay, proper sections!

File: about/alums.md (hunk 2 above)
Comment (icco) [inline about/alums.md]: Github render preview looks nicer with them.

**Comment by mgius** [comment]
:+1:

File: about/alums.md (hunk 4 above)
Comment (icco) [inline about/alums.md]: It'll autolink because of GFM, but it's a good idea.

File: about/alums.md (hunk 5 above)
Comment (icco) [inline about/alums.md]: whoops, good call.

File: README.md (hunk 1 above)
Comment (icco) [inline README.md:8]: Yeah, I wasn't going to specify a ruby, but I started seeing bugs with redcarpet on 2.2 versions of ruby, so just decided to go full hog and update everything to be shiny.

File: about/alums.md (hunk 2 above)
Comment (coyotebush) [inline about/alums.md]: Huh, I don't actually see a difference in the rendering [1](https://github.com/CPLUG/cplug.org/blob/cca68bdbc9bcbf0d541123c6a111a20091421caa/about/alums.md) [2](https://github.com/icco/cplug.org/blob/1328f63b724e2bc665aedf5b26a217e9d06d9ddd/about/alums.md)

**Comment by icco** [comment]
Thanks for the review! And @mgius, I look forward to your addition. @xiongchiamiov and others should get their jam in here too!

File: about/alums.md (hunk 4 above)
Comment (coyotebush) [inline about/alums.md]: Yeah, I don't see it in the Jekyll-generated site. (http://cplug.org is not hosted on GitHub Pages).

**Comment by coyotebush** [comment]
The page is long enough that I'm wondering whether it might even make sense to have a file-per-alum, probably using a Jekyll [collection](https://jekyllrb.com/docs/collections/)?

**Comment by icco** [comment]
Not a bad idea, but I'd say start with this, and then split into a collection if you feel inclined ;)

File: about/alums.md (hunk 2 above)
Comment (icco) [inline about/alums.md]: oh, wow, they must have fixed that bug.

File: _config.yml [hunk 7]
```diff
@@ -1,12 +1,14 @@
...
   - jekyll-paginate
 
-markdown: redcarpet
+markdown: kramdown
+kramdown:
+  input: GFM
+  toc_levels: 1..2
 logo: false
 paginate: 5
-baseurl: /
```
Comment (coyotebush) [inline _config.yml:12]: Our templates (Kasper) have links to `site.baseurl` which no longer work :frowning:

File: about/alums.md [hunk 8]
```diff
@@ -4,206 +4,201 @@ author: mgius
...
-** Other Tidbits:**
-
-Of course, like all true geeks, I hate the CTRL key. Therefore I am a strong VI advocate. I haven&#8217;t lost the music nerd in me either, and I still play trumpet. I also love to travel the world and visit new places. Although I&#8217;m not crazy like Matt, so I don&#8217;t strap on crampons and go climbing giant hunks of ice when I get there.
+### Other Tidbits
 
-\___\___\___\___\___\___\___\___\____
+Of course, like all true geeks, I hate the CTRL key. Therefore I am a strong VI advocate. I haven't lost the music nerd in me either, and I still play trumpet. I also love to travel the world and visit new places. Although I'm not crazy like Matt, so I don't strap on crampons and go climbing giant hunks of ice when I get there.
 
-**[Neil Souza][]** ([nrser](https://github.com/nrser)) &#8211; 2007
+## [Neil Souza][] ([nrser](https://github.com/nrser)) -- 2007
```
Comment (coyotebush) [inline about/alums.md]: Hmm, this results in nested `<a>` tags in the ToC, and browser behavior seems to be to drop the outer one. @nrser maybe we can put your links just below the heading or something?

**Comment by coyotebush** [comment]
Overall I think we can :shipit:, but I'll leave this open for a bit for any other input.
I guess I won't complain at this point about having two rather different changes in one PR.
Thanks @icco!

File: about/alums.md (hunk 3 above)
Comment (brotatos) [inline about/alums.md]: Is this working as intended?

screen shot 2016-03-18 at 12 40 52 am

**Comment by brotatos** [comment]
@OMGTallMonster @frewsxcv @xiongchiamiov  @jhladky 
Would any of you like to add your info as well? Figured we might as well update the alumni page with this  pull open.

**Comment by jhladky** [comment]
Sure, I'll write up something in a few days if that's all right

**Comment by icco** [comment]
Lets save people adding their additional info for future PRs.

**Comment by frewsxcv** [comment]
Yes, but I can just do it in a different PR

**Comment by coyotebush** [comment]
This page could also really do with dates on the entries themselves, since most of them aren't actually very "Current". It might not be straightforward to determine the correct dates, so that can be left for later.

File: about/alums.md [hunk 9]
```diff
@@ -211,4 +206,21 @@ Language: Ruby for tools, scripts and personal stuff. isomorphic Javascript for
 
  [1]: http://suckless.org/
  [2]: http://xkcd.com/
- [3]: http://ruby-lang.org/
\ No newline at end of file
+ [3]: http://ruby-lang.org/
```
Comment (coyotebush) [inline about/alums.md:209]: These links belong to ceeKay's entry and were left at the bottom in the WordPress transition.

**Comment by coyotebush** [comment]
It's live! http://cplug.org/about/alums/

**Comment by nrser** [comment]
hey... are you guys running Ruby 2.3.0 on the production server? and using the Gemfile there (via `bundle exec jekyll ...`? the ruby version and gem versions in `Gemfile` and `Gemfile.lock` should **exactly** match what's being used there.

the easy way to do this is to use `bundle install ...` and `bundle exec ...` on that server to run the actual site.

i haven't read through much of the conversations here, so sorry if i'm just out of the loop on this stuff.

**Pull request merged**